framework/encode/custom_vulkan_api_call_encoders.cpp

```diff
@@ -652,24 +653,39 @@ VKAPI_ATTR VkResult VKAPI_CALL CreateRayTracingPipelinesKHR(VkDevice
         omit_output_data = true;
     }
 
-    auto encoder = VulkanCaptureManager::Get()->BeginTrackedApiCallCapture(
-        format::ApiCallId::ApiCall_vkCreateRayTracingPipelinesKHR);
-    if (encoder)
+    auto device_wrapper = GetWrapper<DeviceWrapper>(device);
+    if (result != VK_OPERATION_DEFERRED_KHR)
```

Comment:
Thank you, @mizhen , for fixing it. I didn't realize the `TrackRayTracingShaderGroupHandles` issue for the deferred case. This branch looks good to me. Could you please add a warning message for delaying writing the block of `vkCreateRayTracingPipelinesKHR` to the capture file? I think the user needs to know it.